src/test/java/org/jenkinsci/plugins/jsch/JSchSSHPublicKeyAuthenticatorTest.java

```diff
@@ -106,21 +106,59 @@ public String getDisplayName() {
+                        "kzSMTHZ9orLB9bthjMTsKmQUduPafqo1AspB0jxcKShias+D4DcymIbXpHEZ0TQZ\n" +
+                        "VjcPMzH4DTnoIaGkyVLg49hEb2ogH9RmM/Hsx7aZA00Aqn/BMDWhkMgHJRActKLC\n" +
+                        "SB0amuNCEu55+bHe5ejB+BhcMhlemtBxliqsfEy52JVwOBTX5sUCggEBAPI9p64n\n" +
+                        "J3tUTsuVWq46SgTJcpqY5GbNPg+f6qHtZgWyoAkAVRgYu7OXLFbcTtBA0axo0pm7\n" +
+                        "3kQCXDEAznHeGAENtodoVHzbvKTmBUJ0WbBqNiJFx8R4u3RB1K4BW5uDo5QTb/aQ\n" +
+                        "CDQ2yF5J60ltdskyfDUahkW2wPJEUnslyeViGgVrP21u0EhC18NpghWUQsN1RQdW\n" +
+                        "e+h0ds15armLMGxxf62ZI5GZl+RLwwjIczjxGm00C++yekasGWnkDikx50UGthyR\n" +
+                        "k4zG/a2ctt00hhgbHafQGwJj3VhTTPZvaqfdcHZYUH1B9S5ZsBHsiadON8w7z2mB\n" +
+                        "aAK/ktrUxe+C7AMCggEBAOzlEAxmfAyg7AUbN516DG1cZ5xX3vWc4s7jy6GsaD7d\n" +
+                        "5OEgZQQVZmi1xcjZaiF/BKOnzFDr/RlQJhNFsT4qotfk5Y79PMed/eoOg7DqdxAa\n" +
+                        "w1uzlqhrDHnlBXAXY5De7L3su4rBMg2O570L5FUbffPgym4aXaxg/etVUV7pQqPV\n" +
+                        "lcuyxBp93lG6MnHTJ4CZ7gTwb+XyFfhdF6P9WZOwpdoCjQg2eoYMZBo5h7LivziY\n" +
+                        "7N6hKgm5EQ2mrfJqHB3H540Dy4P03LEjk5xQDUnBn/gpj4ZePBFaiueEvlOXiCwP\n" +
+                        "+jXIfF+kAlef/UEBOJUHzzp39elqkU0msKLxghLzlU8CggEBAOrcBBrBM3JLRyny\n" +
+                        "4DxTnzgM5+QjoC5bh3Q2o5HjTSrxCGAxxk7ajAGO7Bo69t7KOX9jEeyjTNe7Qg7w\n" +
+                        "rTeREMzUsseNy3xSvw9RRIAttldoYpvP8+L0+ym4Oa+K+XpJousJ/V+cPZgCFTn1\n" +
+                        "iP1j9+sR24LQ+KXWjjNVMnLbLGgNORVP6er7qUymIfL/9HNfj2tZ41c97lxtrlGB\n" +
+                        "Coxh+szpLdTtyKJ9u9pH6gw17CClAe4mq/v1mr+yU+FqjqA6FfPCkgYYzmmK9KDC\n" +
+                        "dDj7l5b/kz0Ec2tZz1y3RsMXOt0NwN+8uCz5KfGKWz7FiqB/IXIN+wZbxLAFdShd\n" +
+                        "aprQ4GkCggEBAIOKokGwertsdAJV3aj1B4eGYwYeiPCrgAnP1dfdazlVb21O1qjQ\n" +
+                        "1T/Zh40CpPsak9HoL/zTPYRby/ixnzzc4fWt5YZjuedCJKdeDeQkHZ70rXvzGfpF\n" +
+                        "DvV0pXNbmW7tSlof5PekVY3Px4Bi5RQZIvRT4zQGMfOxG+4cPwXL0rQ0umwUxO3M\n" +
+                        "7LFHChHIZUv0rYVSmV/+8BIsZx8pZB8tXLrU5ckkrx5WLROe1GoRnIrp58WrijNB\n" +
+                        "72U8I6TTJO+ofDwCWnTYd99o2ONYVDibap+bPFYpZ4NfWng6bpDuOK/240IQJHfb\n" +
+                        "E23iqfb5nZircHeP+x30jeBgVn70Sf0KAuMCggEBAOG5wh1m5H981ZY1EPFxYpvH\n" +
+                        "mh2K0eoNUGlSmWcuq+L1k93xG5bsaKpZH2xSpRqcwARvAy+0kIncE14QAVsBuse9\n" +
+                        "B4G86IUlsKYe2aZt3Zo+2bz+CdHWZHYcNyj+aYrzRwB/z3d3F5sYccPL/Tmu34KQ\n" +
+                        "tU4Cglzfz8lGzpIwM7HN2JyubjO0Iy1UbEqP0RqK38LYP0SYPoinXVKXKAwEZ7gp\n" +
+                        "23bCUz4li9GjJ/Ke7ztcFES/9HzyUTw/L7VS6LHf99W0yjHpZ2OaqxrSXHGSXP0R\n" +
+                        "GNuyG1OI+KmyNhb332Bm3R8mSM3mw9wBrFM1VUmBtNwjn1P8qnyEwzo9tdZNxoI=\n" +
+                        "-----END RSA PRIVATE KEY-----";
```

Comment:
Maybe the test failures are because you need a trailing newline at the end of the String? Not sure, but the old key had one, so seems worth trying.

Reply:
Stderr for the failing tests points to the newer cores not being happy with using SHA1, maybe after the bump to 4096 bit keys: `java.security.InvalidKeyException: The security strength of SHA-1 digest algorithm is not sufficient for this key size`.

Reply:
Oh damn it, it's still failing? That was a similar error for the old RSA key that was in this test. Might need to do some more digging.

Reply:
I figured it out. It was due to the use of `SimpleGeneratorHostKeyProvider` from MINA without specifying non-default key parameters. The default is too weak for JSch to accept nowadays. Updating the PR with a fix.